VictoriaAlbert/VictoriaAlbert/VictoriaAndAlbertMuseumTableViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  VictoriaAndAlbertMuseumTableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by John Gabriel Breshears on 11/10/16.
+//  Copyright © 2016 John Gabriel Breshears. All rights reserved.
+//
+
+import UIKit
+
+class VictoriaAndAlbertMuseumTableViewController: UITableViewController {
+    var searchKey = "ring"
+    
+    
+// Okay so museum now has a shit ton of values. It contains all the object,datetex,place,title,primayImageId, and location data for all ring search results.
+    var museum: [Item] = []
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        loadData()
+        
+
+    }
+
+    
+    func loadData() {
+        
+        
+        
+        
+        
+        APIRequestManager.manager.getData(searchKey: searchKey) { (data: Data?) in
+            if data != nil {
+                if let validData = Museum.getMuseumData(from: data!) {
+                    DispatchQueue.main.async {
+                        self.museum = validData
+                    }
+                }
+                    
+                }
+            }
+        }
+    
+    
+//    func loadSmallPicture() {
+//        APIRequestManager.manager.getLittlePicture(primaryImageId:  callback: <#T##(Data?) -> Void#>)
+//    }
+
+    
+    
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return museum.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "oldStuffImeanHistory", for: indexPath)
+        
+        let m = museum[indexPath.row]
+        
+ // Why the fuck does the below code not worK? Damn, i need to review table views....
+cell.textLabel?.text = m.title
```

Comment:
The second reason you weren't seeing data is that the ```title``` field was actually very spotty. Other fields like ```location``` were more reliable and this is why I prescribed setting the textLabel to a combination of object, location and date. Once I made these two changes I did see data load in your table.